src/Kdyby/Annotations/DI/AnnotationsExtension.php

```diff
@@ -81,7 +82,7 @@ public function getConfig(array $defaults = NULL, $expand = TRUE)
 			$config = Nette\DI\Config\Helpers::merge($config, ['ignore' => $globalConfig['doctrine']['ignoredAnnotations']]);
 		}
 
-		return $this->compiler->getContainerBuilder()->expand($config);
+		return Nette\DI\Helpers::expand($config, $this->getContainerBuilder()->parameters);
```

Comment:
Indeed. But there is another way to do it now.

https://api.nette.org/2.3/source-Bridges.ApplicationDI.ApplicationExtension.php.html#40-48

```
extensions:
    nette.application: Nette\Bridges\ApplicationDI\ApplicationExtension( %debugMode% )
```